apps/pie-microsite/src/assets/styles/components/_notification.scss

```diff
@@ -1,11 +1,12 @@
 @use '../fozzie-config' as f;
 
+$combinedIconAndPaddingWidth:       calc($icon-width + $notification-padding);
```

Comment:
I didn't think it would be happy with this using other SCSS variables before they're defined?

Reply:
You're right. It errors (i skipped the hook). Reverted.